packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1272,3 +1353,68 @@ class _EffectiveMouseCursor extends MaterialStateMouseCursor {
   @override
   String get debugDescription => 'MaterialStateMouseCursor(PopupMenuItemState)';
 }
+
+class _DefaultsM2 extends PopupMenuThemeData {
+  _DefaultsM2(this.context)
+    : super(
+      elevation: 8.0,
+      iconSize: _kDefaultIconSize,
+    );
+
+  final BuildContext context;
+  late final ThemeData _theme = Theme.of(context);
+  late final TextTheme _textTheme = _theme.textTheme;
+
+  @override
+  TextStyle? get textStyle => _textTheme.subtitle1;
+}
+
+// BEGIN GENERATED TOKEN PROPERTIES
+
+// Generated code to the end of this file. Do not edit by hand.
+// These defaults are generated from the Material Design Token
+// database by the script dev/tools/gen_defaults/bin/gen_defaults.dart.
+
+// Generated version v0_98
+class _TokenDefaultsM3 extends PopupMenuThemeData {
+  _TokenDefaultsM3(this.context)
+    : super(
+      elevation: 3.0,
+      dividerThickness: 1.0,
+      iconSize: 24.0,
+    );
+
+  final BuildContext context;
+  late final ThemeData _theme = Theme.of(context);
+  late final ColorScheme _colors = _theme.colorScheme;
+  late final TextTheme _textTheme = _theme.textTheme;
+
+  @override MaterialStateProperty<TextStyle?>? get labelTextStyle {
+    return MaterialStateProperty.resolveWith((Set<MaterialState> states) {
+    final TextStyle style = _textTheme.labelLarge!;
+      if (states.contains(MaterialState.disabled)) {
+        return style.apply(color: _colors.onSurface.withOpacity(0.38));
+      }
+      return style.apply(color: _colors.onSurface);
+    });
+  }
+
+  @override
+  Color? get color => _colors.surface;
+
+  @override
+  Color? get shadowColor => _colors.shadow;
+
+  @override
+  Color? get surfaceTintColor => _colors.surfaceTint;
+
+  @override
+  Color? get iconColor => _colors.onSurfaceVariant;
+
+  @override
+  Color? get dividerColor => _colors.surfaceVariant;
+
+  @override
+  ShapeBorder? get shape => const RoundedRectangleBorder(borderRadius: BorderRadius.only(topLeft: Radius.circular(4.0), topRight: Radius.circular(4.0), bottomLeft: Radius.circular(4.0), bottomRight: Radius.circular(4.0)));
```

Comment:
We should regenerate the defaults with the latest gen_defaults code from master as the shape code will optimize this down to a single parameter.